components/runtime/openjdk-8/Makefile

```diff
@@ -12,53 +12,124 @@
 #
 # Copyright 2016 Alexander Pyhalov
 # Copyright 2019 Michal Nowak
+# Copyright 2022 Daniel Bell
 #
 
+BUILD_STYLE=configure
+BUILD_BITS=64
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		openjdk
-OPENJDK_BUILD_NUMBER=	232
-COMPONENT_VERSION=	jdk8u-jdk8u$(OPENJDK_BUILD_NUMBER)-b09.1
-COMPONENT_REVISION=	3
+COMPONENT_NAME=         openjdk
+OPENJDK_BUILD_NUMBER=	352
+COMPONENT_VERSION=      jdk8u-jdk8u$(OPENJDK_BUILD_NUMBER)-ga
 IPS_COMPONENT_VERSION=	1.8.$(OPENJDK_BUILD_NUMBER)
```

Comment:
It would be great if you could convert this (together with `COMPONENT_VERSION` above) to `COMPONENT_VERSION` + `HUMAN_VERSION`.